react/components/SliderContext.tsx

```diff
@@ -96,6 +96,7 @@ export interface SliderLayoutProps {
   centerMode?: ResponsiveValuesTypes.ResponsiveValue<
     'center' | 'to-the-left' | 'disabled'
   >
+  centerModeSlidesGap?: number | undefined
```

Comment:
If you don't need explicit undefined value, look this suggestion
```suggestion
  centerModeSlidesGap?: number
```